core/lib/globals_shared.h

```diff
@@ -2017,4 +2017,30 @@ typedef struct _priv_mcontext_t {
 #include "mcxtx.h"
 } priv_mcontext_t;
 
+/* DR_API EXPORT BEGIN */
+/**
+ * Encodings of an application's argument.
+ */
+typedef enum {
+    APP_ARG_ASCII,
```

Comment:
To avoid namespace conflcts when linked statically into apps, let's follow the convention of having a DR_ prefix on new global symbols.

Reply:
Done.